### PR TITLE
Clarified Secrets file mode

### PR DESCRIPTION
**Pull request by jqmichael**
1. The example is using YAML which supports octal notation. Updated the corresponding doc. 
2. Clarified we need to follow the symlink to find the correct file mode.

### CODE REVIEW

File: content/en/docs/concepts/configuration/secret.md [hunk 1]
```diff
@@ -580,16 +580,35 @@ spec:
...
+total 0
+lrwxrwxrwx 1 root root 15 May 18 00:18 password -> ..data/password
+lrwxrwxrwx 1 root root 15 May 18 00:18 username -> ..data/username
+
+# cd /etc/foo/..data
+# ls -l
+total 8
+-r-------- 1 root root 12 May 18 00:18 password
+-r-------- 1 root root  5 May 18 00:18 username
+```
```
Comment (sftim) [inline content/en/docs/concepts/configuration/secret.md:610]: Kubernetes documentation usually [separates commands from their output.](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output) Please update this to match that style.

**Comment by jqmichael** [comment]
@sftim 

Appreciated the reference. Force pushed the fix. Please let me know if there's anywhere else I need to fix.

File: content/en/docs/concepts/configuration/secret.md [hunk 2]
```diff
@@ -590,6 +590,38 @@ Note that the JSON spec doesn't support octal notation, so use the value 256 for
...
+
+The output is similar to this:
+```
+total 0
+lrwxrwxrwx 1 root root 15 May 18 00:18 password -> ..data/password
+lrwxrwxrwx 1 root root 15 May 18 00:18 username -> ..data/username
+```
+
+Follow the simlink to find the correct file mode.
```
[the remark this answers is not in the document]
Comment (jqmichael) [reply content/en/docs/concepts/configuration/secret.md]: Thanks for the catch. Fixed.

File: content/en/docs/concepts/configuration/secret.md [hunk 3]
```diff
@@ -590,6 +590,38 @@ Note that the JSON spec doesn't support octal notation, so use the value 256 for
...
+
+```
+cd /etc/foo/..data
+ls -l
```
[the remark this answers is not in the document]
Comment (jqmichael) [reply content/en/docs/concepts/configuration/secret.md:615]: Indeed there're more than one symlink involved.  Didn't output here for the sake of simplicity. The original intention is to let reader understand the file mode is set up correct if they follow the symlink. 

Does that make sense?

File: content/en/docs/concepts/configuration/secret.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jqmichael) [reply content/en/docs/concepts/configuration/secret.md]: @sftim

File: content/en/docs/concepts/configuration/secret.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (sftim) [reply content/en/docs/concepts/configuration/secret.md:615]: Sure, no worries!

**Comment by sftim** [comment]
This is ready for an approver to add /approve, at which point Prow / Tide will merge it.

It's also ready for another /lgtm: I'm an approver but don't want to do both reviews here. If someone else adds /lgtm because they're happy with this, I can /approve.

**Comment by jqmichael** [comment]
@sftim Any chance you can approve?

**Comment by sftim** [comment]
Happy to
/lgtm

**Pull request merged**